diff -Nur geki5-KXL-0.3.0.orig/configure.in geki5-KXL-0.3.0/configure.in
--- geki5-KXL-0.3.0.orig/configure.in	Mon Sep  3 16:28:48 2001
+++ geki5-KXL-0.3.0/configure.in	Mon May 12 14:44:52 2003
@@ -29,11 +29,18 @@
         BMP_PATH=$prefix/share/geki5/bmp
         WAV_PATH=$prefix/share/geki5/wav
 fi
+if test "x$localstatedir" = "xNONE"; then
+	SCORE_PATH=/var/games
+else
+	SCORE_PATH=$localstatedir/games
+fi
 AC_DEFINE_UNQUOTED(DATA_PATH,"$DATA_PATH")
+AC_DEFINE_UNQUOTED(SCORE_PATH,"$SCORE_PATH")
 AC_DEFINE_UNQUOTED(BMP_PATH,"$BMP_PATH")
 AC_DEFINE_UNQUOTED(WAV_PATH,"$WAV_PATH")
 AC_DEFINE_UNQUOTED(TITLE,"$PACKAGE $VERSION")
 AC_SUBST(DATA_PATH)
+AC_SUBST(SCORE_PATH)
 AC_SUBST(BMP_PATH)
 AC_SUBST(WAV_PATH)
 
diff -Nur geki5-KXL-0.3.0.orig/data/Makefile.am geki5-KXL-0.3.0/data/Makefile.am
--- geki5-KXL-0.3.0.orig/data/Makefile.am	Mon Sep  3 17:06:34 2001
+++ geki5-KXL-0.3.0/data/Makefile.am	Mon May 12 14:45:29 2003
@@ -4,5 +4,6 @@
 EXTRA_DIST = stage1.dat stage2.dat stage3.dat stage4.dat stage5.dat\
 	     stage6.dat stage7.dat stage8.dat .score
 
-install:
-	$(INSTALL) -c -m 666 ./.score $(DESTDIR)$(DATA_PATH)
+install-data-local:
+	$(INSTALL) -d -m 755 $(DESTDIR)$(SCORE_PATH)
+	$(INSTALL) -c -m 666 ./.score $(DESTDIR)$(SCORE_PATH)/geki5.score
diff -Nur geki5-KXL-0.3.0.orig/src/ranking.c geki5-KXL-0.3.0/src/ranking.c
--- geki5-KXL-0.3.0.orig/src/ranking.c	Mon Sep  3 15:20:55 2001
+++ geki5-KXL-0.3.0/src/ranking.c	Mon May 12 14:45:50 2003
@@ -36,7 +36,7 @@
   FILE *fp;
   Uint16 i;
 
-  if ((fp = fopen(DATA_PATH "/.score", "r"))) {
+  if ((fp = fopen(SCORE_PATH "/geki5.score", "r"))) {
     fscanf(fp, "%d", &(Root->HiScore));
     for (i = 0; i < 5; i ++)
       fscanf(fp, "%d %d %s",
@@ -65,7 +65,7 @@
   FILE *fp;
   Uint16 i;
 
-  if ((fp = fopen(DATA_PATH "/.score", "w"))) {
+  if ((fp = fopen(SCORE_PATH "/geki5.score", "w"))) {
     fprintf(fp, "%d\n", Root->HiScore);
     for (i = 0; i < 5; i ++)
       fprintf(fp, "%d %d %s\n",
